scripts/deploy_subscription_manager.py

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/python3
-from brownie import Contract, SubscriptionManager, Wei, project
-
+from ape import project
 from scripts.utils import get_account
+from web3 import Web3
 
-INITIAL_FEE_RATE = Wei("1 gwei")
+INITIAL_FEE_RATE = Web3.to_wei(1, "gwei")
 
 
 def main(id=None):
     deployer = get_account(id)
-    oz = project.load("OpenZeppelin/openzeppelin-contracts@4.5.0/")
+    dependency = project.dependencies["openzeppelin"]["4.8.1"]
+    admin = dependency.get("ProxyAdmin")
+    proxy = dependency.get("TransparentUpgradeableProxy")
```

Comment:
@fubuloubu Is there a better way to deploy contracts from a dependency (e.g., like an OZ proxy here?)

Reply:
This should work:

```py
oz = project.dependencies["openzeppelin"]["4.8.1"]
deployer.deploy(oz.ProxyAdmin)
```

Reply:
✔️